SQL/0000-00-00-schema.sql

```diff
@@ -69,8 +71,9 @@ CREATE TABLE `language` (
   UNIQUE KEY (`language_code`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
-INSERT INTO language (language_code, language_label) VALUES
-    ('en-CA', 'English');
+INSERT INTO `language` (`language_code`, `language_label`) VALUES
+    ('en-CA', 'English'),
+    ('fr-CA', 'Français');
```

Comment:
We should probably remove the `-CA` for now IMO, as we will very likely not use it in the translations. I am not sure how that plays out with other tables and modules that refer to the languages though.